UPGRADE.md

```diff
@@ -10,6 +10,12 @@ Method `Doctrine\ORM\AbstractQuery#useResultCache()` which could be used for bot
 To optimize DB interaction, `Doctrine\ORM\Tools\Pagination\Paginator` no longer fetches identifiers to be able to
 perform the pagination with join collections when max results isn't set in the query.
 
+## Minor BC BREAK: tables filtered with `schema_filter` are no longer created
```

Comment:
I would consider this a bugfix, and not a BC break.

Reply:
https://github.com/doctrine/orm/pull/7875#issuecomment-546242165 and https://github.com/doctrine/orm/pull/7875#issuecomment-546615794 give a bit more explanation on whey we also consider this a behavioural BC-break.